### DISCUSSION TITLE
freeones scrapes not complete

### DISCUSSION THREAD

**Discussion by plonk-droid** [stashapp/stash#3841] [Q&A]
I used to get a lot of information when scraping a performer with the default freeones scraper. Things like gender, country, aliases etc.
These days I seem to only get the bio details, picture and some random other stuff like ethnicity, while freeones website has the other information aswell.
I'm not sure when exactly this behaviour changed. I moved through a lot of stash versions (currently on 0.21.0 on centos) and all the databases migrations seemed to have gone well to me.
Does this freeones scraper behaviour ring any bell with anyone? Is there something I can do to get it to work like before?
Thanks for your feedback.

**Comment by Charlotte-br560** [comment]
It seems like the Freeones scraper isn't fetching all the information it used to. You could check for updates in the project's repository. Additionally, consider trying [Crawlbase](https://crawlbase.com/) as an alternative.

**Comment by feederbox826** [comment]
Unable to replicate it on the latest version, but either way belongs in https://github.com/stashapp/CommunityScrapers/issues if similar issues arise
